contracts/0.4.24/Lido.sol

```diff
@@ -123,9 +123,9 @@ contract Lido is ILido, IsContract, Pausable, AragonApp {
     /**
     * @notice Send funds to the pool with optional _referral parameter
     * @dev This function is alternative way to submit funds. Supports optional referral address.
-    * @return StETH Amount of StETH tokens generated
+    * @return Amount of StETH shares generated
     */
-    function submit(address _referral) external payable returns (uint256 StETH) {
+    function submit(address _referral) external payable returns (uint256) {
```

Comment:
this behavior should be covered by test and returned amount should be asserted.

Reply:
Discussed verbally, tests will be later.

Reply:
Reason: return value of transact is not visible in the returned object of promise. To test this we need to wrap into `.call()`able mock but we are under time pressure. We'll put this in backlog.